src/GraphQL/Language/AST/ValueNodes/ObjectValue.cs

```diff
@@ -54,9 +56,9 @@ public object Value
         {
             get
             {
-                var obj = new Dictionary<string, object>(ObjectFieldsList.Count);
+                var obj = new Dictionary<string, object?>(ObjectFieldsList.Count);
                 foreach (var item in ObjectFieldsList)
-                    obj.Add(item.Name, Field(item.Name).Value.Value);
```

Comment:
Do you have thoughts, why was it so written here?

Reply:
Nope.  It made no sense to me, so I fixed it.